src/schema/__tests__/sale_artwork.test.js

```diff
@@ -394,12 +395,21 @@ describe("SaleArtwork type", () => {
           cents: 420000,
           display: "€4,200",
         },
+        {
```

Comment:
This was the main logic change. I'm trying to make it so that when a client pulls `increments(useMyMaxBid: true)`, they can by default show the first item in the list. If we didn't do this, they'd have to still have some custom logic for when the increments are calculated based on your max bid (and to show the second item in the array in that case).